gui/wxpython/tplot/frame.py

```diff
@@ -212,7 +212,7 @@ def _layout(self):
             self.coorval = gselect.CoordinatesSelect(
                 parent=self.controlPanelRaster, giface=self._giface
             )
-        except:
+        except (AttributeError, TypeError):
```

Comment:
I think this is trying to capture NotImplementedError, see wxpython/core/giface.py